dashboard/config/routes.rb

```diff
@@ -235,6 +235,7 @@
     member do
       get '/:filename', to: 'level_starter_assets#file'
       post '', to: 'level_starter_assets#upload'
+      delete '/:filename', to: 'level_starter_assets#destroy'
```

Comment:
this can be replaced by specifying `only: [..., :destroy]` above

Reply:
do you mean on line 234? because that would result in `DELETE /level_starter_assets/:level_name`, but we want `DELETE /level_starter_assets/:level_name/:filename`

Reply:
yes, I did mean on line 234, but only if you were going to go ahead with changing the routes around to just use `resoucres` and not use `member`. since we're punting on that, there is nothing else to do here right now.